tools/attribute.js

```diff
@@ -5,6 +5,15 @@ const path = process.argv[3] || '../package.json';
 const attributes = require(path);
 const value = attributes[attributeName];
 
-if (value !== undefined) {
-  console.log(value);
+function main() {
+  if (process.argv[4]) {
```

Comment:
nit: naming this would be helpful. Seems like this is an enumeration that specifies a templating mode?